common4j/src/main/com/microsoft/identity/common/java/util/ported/LocalBroadcaster.java

```diff
@@ -59,16 +62,19 @@ public void unregisterCallback(@NonNull final String alias){
         mReceivers.remove(alias);
     }
 
-    public void broadcast(@NonNull final String alias, @NonNull final PropertyBag propertyBag){
+    public void broadcast(@NonNull final String alias, @NonNull final PropertyBag propertyBag) {
         final String methodName = ":broadcast";
-        final IReceiverCallback receiver = mReceivers.get(alias);
-
-        if (receiver != null){
-            Logger.info(TAG + methodName, "broadcasting to alias: " + alias);
-            receiver.onReceive(propertyBag);
-        } else {
-            Logger.info(TAG + methodName, "No callback is registered with alias: " + alias +
-                    ". Do nothing.");
-        }
+        sBroadcastExecutor.execute(new Runnable() {
```

Comment:
Is it possible to generate some waiting-time metrics here, just in case something gets stuck in queue for the executor?